extensions/metrics-cluster-feature/renderer.tsx

```diff
@@ -1,21 +1,55 @@
-import { LensRendererExtension } from "@k8slens/extensions";
+import { LensRendererExtension, Store, Interface, Component } from "@k8slens/extensions";
 import { MetricsFeature } from "./src/metrics-feature";
-import React from "react";
 
 export default class ClusterMetricsFeatureExtension extends LensRendererExtension {
-  clusterFeatures = [
-    {
-      title: "Metrics Stack",
-      components: {
-        Description: () => (
-          <span>
-            Enable timeseries data visualization (Prometheus stack) for your cluster.
-            Install this only if you don&apos;t have existing Prometheus stack installed.
-            You can see preview of manifests <a href="https://github.com/lensapp/lens/tree/master/extensions/metrics-cluster-feature/resources" rel="noreferrer" target="_blank">here</a>.
-          </span>
-        )
-      },
-      feature: new MetricsFeature()
+  onActivate() {
+    const category = Store.catalogCategories.getForGroupKind<Store.KubernetesClusterCategory>("entity.k8slens.dev", "KubernetesCluster");
+
+    if (!category) {
+      return;
+    }
+
+    category.on("contextMenuOpen", this.clusterContextMenuOpen.bind(this));
```

Comment:
This is a mouthful. Maybe eventually we can add a helper method to the `LensExtension` class so it simplifies to something like
```
this->CategoryOn("KubernetesCluster", "contextMenuOpen", this.clusterContextMenuOpen);
```

Reply:
`->`?!?!

Is your c++ memories leaking \s.

Reply:
Oops. My memory is leaky in general. And all languages are looking more and more the same.